addOns/graphql/src/main/java/org/zaproxy/addon/graphql/GraphQlFingerprinter.java

```diff
@@ -61,7 +61,7 @@ public GraphQlFingerprinter(URI endpointUrl) {
     }
 
     public void fingerprint() {
-        Map<String, BooleanSupplier> fingerprinters = new LinkedHashMap<>(29);
+        Map<String, BooleanSupplier> fingerprinters = new LinkedHashMap<>(35);
```

Comment:
Incrementing this was previously missed, I just did it as a clean code effort to prevent re-sizing.

Reply:
It worth reading how the initial capacity of the maps work (not the same as lists).

Reply:
Thanks, that was worth reading. Full honesty I'm not sure I'll remember a few months from now.

So based on default load of 0.75, if we didn't specify then it would be resized at 12, to roughly 32. Then at 24 to roughly 64. As it's being filled. So shall I just declare it as 64, and add a note to increase it if the list ever grows past 48?

Reply:
I'd not bother setting the initial capacity.